__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest");
+const seed= require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const connection = require("../db/connection");
+const app = require("../app");
+
+beforeEach(() => {
+    return seed(data)
+});
+
+afterAll(() => {
+    return connection.end()
+});
+
+describe.only("GET /api/topics", () => {
+  test("200: responds with object of array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({body}) => {
+        expect(body.topics).toHaveLength(3);
```

Comment:
Great length assertion, this is really important when using a forEach which would still falsely pass tests if we passed it an empty array